src/mcMd/potentials/pair/PairFactory.cpp_r

```diff
@@ -81,24 +82,25 @@ namespace McMd
    PairFactory::mdFactory(const std::string& name, System& system) const
    {
       MdPairPotential* ptr = 0;
+      MdSystem& mdsystem = dynamic_cast<MdSystem&>(system);
 
       // Try subfactories first
-      ptr = tryMdSubfactories(name, system);
-      if (ptr) return ptr;
+      //ptr = tryMdSubfactories(name, system);
```

Comment:
Huada, please explain why it was necessary to comment out the ptr = tryMdSubFactories(name, system);
lines at lines 86:87 of mcMd/potentials/pair/PairFactory.cpp_r